flow/layers/clip_path_layer.cc

```diff
@@ -48,7 +48,7 @@ void ClipPathLayer::Paint(PaintContext& context) const {
   TRACE_EVENT0("flutter", "ClipPathLayer::Paint");
   FXL_DCHECK(needs_painting());
 
-  Layer::AutoSaveLayer save(context, paint_bounds(), nullptr);
+  SkAutoCanvasRestore save(&context.canvas, true);
```

Comment:
This won't work. It will cause artifacts.
See this article for a discussion of why: https://master-docs-flutter-io.firebaseapp.com/flutter/dart-ui/Canvas/saveLayer.html